api/handler/trigger.go

```diff
@@ -73,7 +73,9 @@ func getTrigger(writer http.ResponseWriter, request *http.Request) {
 	}
 
 	if err := checkingTemplateFilling(request, *trigger); err != nil {
-		middleware.GetLoggerEntry(request).Warning(err)
+		middleware.GetLoggerEntry(request).Warningb().
+			Value("error_response", err).
```

Comment:
seems can be without _response

Reply:
I chose `error_response` because is is not actually an `error` interface. Is is `api.ErrorResponse` and we already use `error` tag name for actual errors. But I will change it, if you think that it would be better

Reply:
aaaa, ok 
this is wrapper struct